app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
Provide typing

Reply:
Sorry, I can`t understand where do I need typing?) 
Do you mean from typing import Any or something like that?

Reply:
No just:
```suggestion
    alive: list[Animal] = []
```

Reply:
Done